client/js/src/client.ts

```diff
@@ -1180,7 +1188,7 @@ function handle_message(
 	data: any,
 	last_status: Status["stage"]
 ): {
-	type: "hash" | "data" | "update" | "complete" | "generating" | "none";
+	type: "hash" | "data" | "update" | "complete" | "generating" | "log" | "none";
```

Comment:
I was thinking about this for a while, `error` being a separate type from `log` but I think it is correct even though we display them in a consistent manner. `gr.Error()` is relating to the status of the function itself, while `gr.Info()` and `gr.Warning()` are completely unrelated.